fails to build for me bacause of mrkd it looks like:
` $ nix-review pr 144537`
```
builder for '/nix/store/dkivy7sslkc5rr1yb63df6hq25xk3lmz-clightning-0.10.2.drv' failed with exit code 2; last 10 log lines:
  printgen wire/peer_printgen.c
  tools/generate-wire.py --page impl tools/test/test_gen.h test_type < tools/test/test_cases > tools/test/test_gen.c.tmp.c
  printgen wire/channel_type_printgen.c
  echo '#include "test_gen.h"' > tools/test/print_gen.c
  tools/generate-wire.py -P --page impl tools/test/print_gen.h test_type < tools/test/test_cases >> tools/test/print_gen.c
  mrkd doc/lightning-cli.1.md
  /nix/store/phqa311klldrcbwid1i22dwnpfc9dnma-bash-5.1-p8/bin/bash: line 1: mrkd: command not found
  make: *** [doc/Makefile:107: doc/lightning-cli.1] Error 127
  make: *** Waiting for unfinished jobs....
  rm tools/test/test_gen.c.tmp.c
cannot build derivation '/nix/store/g70vq1w5cmfcv9hds2i0qxj0bf7lzds5-review-shell.drv': 1 dependencies couldn't be built
error: build of '/nix/store/g70vq1w5cmfcv9hds2i0qxj0bf7lzds5-review-shell.drv' failed
```

--- a/pkgs/applications/blockchains/clightning/default.nix
+++ b/pkgs/applications/blockchains/clightning/default.nix
@@ -16,15 +16,15 @@
 , zlib
 }:
 let
-  py3 = python3.withPackages (p: [ p.Mako ]);
+  py3 = python3.withPackages (p: [ p.Mako p.mrkd ]);
 in
 stdenv.mkDerivation rec {
   pname = "clightning";
-  version = "0.10.1";
+  version = "0.10.2";
 
   src = fetchurl {
     url = "https://github.com/ElementsProject/lightning/releases/download/v${version}/clightning-v${version}.zip";
-    sha256 = "9271e9e89d60332b66afedbf8d6eab2a4a488782ab400ee1f60667d73c5a9a96";
+    sha256 = "3c9dcb686217b2efe0e988e90b95777c4591e3335e259e01a94af87e0bf01809";
   };
 
   nativeBuildInputs = [ autogen autoconf automake gettext libtool pkg-config py3 unzip which ];

--- a/pkgs/development/python-modules/mrkd/default.nix
+++ b/pkgs/development/python-modules/mrkd/default.nix
@@ -1,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, jinja2
+, mistune
+, pygments
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "mrkd";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "456f8c1be99da268554b29c6b5383532e58119def5a65d85270bc6a0ecc26aaf";
+  };
+
+  propagatedBuildInputs = [ jinja2 mistune pygments setuptools ];
+
+  pythonImportsCheck = [ "mrkd" ];
+
+  meta = with lib; {
+    description = "Write man pages using Markdown, and convert them to Roff or HTML";
+    homepage = "https://github.com/refi64/mrkd";
+    license = licenses.bsd2;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -3321,6 +3321,8 @@ with pkgs;
 
   mq-cli = callPackage ../tools/system/mq-cli { };
 
+  mrkd = with python3Packages; toPythonApplication mrkd;
+
   n2n = callPackage ../tools/networking/n2n { };
 
   nextdns = callPackage ../applications/networking/nextdns { };

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -4874,6 +4874,8 @@ in {
 
   mpyq = callPackage ../development/python-modules/mpyq { };
 
+  mrkd = callPackage ../development/python-modules/mrkd { };
+
   ms-cv = callPackage ../development/python-modules/ms-cv { };
 
   msal = callPackage ../development/python-modules/msal { };

